server/src/main/java/org/opensearch/rest/action/admin/cluster/RestQueryGroupStatsAction.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest.action.admin.cluster;
+
+import org.opensearch.action.admin.cluster.wlm.QueryGroupStatsRequest;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.core.common.Strings;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestActions;
+
+import java.io.IOException;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.unmodifiableList;
+import static org.opensearch.rest.RestRequest.Method.GET;
+
+/**
+ * Transport action to get QueryGroup stats
+ *
+ * @opensearch.experimental
+ */
+public class RestQueryGroupStatsAction extends BaseRestHandler {
+
+    @Override
+    public List<Route> routes() {
+        return unmodifiableList(asList(new Route(GET, "/_wlm/query_group_stats"), new Route(GET, "/_wlm/query_group_stats/{nodeId}")));
+    }
```

Comment:
Since this is core OS API we should remove the `_wlm` and keep it simple `query_group/stats` and it would be good if we could add queryQroupId as filter and other as mentioned in the stats issue: https://github.com/opensearch-project/OpenSearch/issues/15120

Reply:
Are you suggesting to turn `query_group/stats/{nodeId}` into `query_group/stats?nodeId=` and also support `query_group/stats?breach=`?